plugin/api.go

```diff
@@ -334,11 +334,16 @@ type API interface {
 	// Minimum server version: 5.10
 	SearchPostsInTeam(teamId string, paramsList []*model.SearchParams) ([]*model.Post, *model.AppError)
 
-	// AddChannelMember creates a channel membership for a user.
+	// AddChannelMember joins a user to a channel (as if they joined themselves)
 	//
 	// Minimum server version: 5.2
 	AddChannelMember(channelId, userId string) (*model.ChannelMember, *model.AppError)
 
+	// AddUserToChannel adds a user to a channel as a specified user.
```

Comment:
I think this convention ("as a specified user" -- referring to the `userRequestorId`/`asUserId` in the api call) is confusing in the documentation. Can we clarify what it means here in the comment?  I think it would help community members (and new staff).

Reply:
Perhaps, `adds a user to a channel (as if the specified user had invited them)`